src/theory/quantifiers/sygus_inst.cpp

```diff
@@ -190,6 +381,13 @@ void SygusInst::preRegisterQuantifier(Node q)
   addCeLemma(q);
 }
 
+void SygusInst::ppNotifyAssertions(const std::vector<Node>& assertions)
+{
+  d_notified_assertions.clear();
```

Comment:
This actually needs to be a user-context dependent list, where we don't clear and only add here.  This will ensure sygus-inst does the right thing in incremental mode.

Reply:
(Which also assumes sygus will eventually be available in incremental mode...) :)

Reply:
done, it's not a CDHashSet.